### PR TITLE
add logout button

### PR DESCRIPTION
**Pull request by berezh**

### CODE REVIEW

File: example/src/react-barong/index.js [hunk 1]
```diff
@@ -41,22 +44,27 @@ var InputError = function (_a) {
...
+            BarongApiUtil.login(host, data)
+                .then(function (response) {
+                if (response.status === 200) {
+                    window.location.replace(redirection);
+                }
+                else {
+                    window.console.error(response);
+                }
+            })
+                .catch(function (err) {
```
Comment (akhlopiachyi) [inline example/src/react-barong/index.js:63]: fix the indentation

File: example/src/react-barong/index.js [hunk 2]
```diff
@@ -79,17 +87,24 @@ var BarongLoginForm = function (_a) {
...
+            window.location.replace(redirection);
+        }
+        else {
+            BarongApiUtil.register(host, data)
+                .then(function (response) {
+                response.status === 201
+                    ? window.location.replace("" + redirection)
+                    : window.console.error(response);
+            })
+                .catch(function (error) {
```
Comment (akhlopiachyi) [inline example/src/react-barong/index.js:103]: fix indentation

File: example/src/react-barong/index.js [hunk 3]
```diff
@@ -79,17 +87,24 @@ var BarongLoginForm = function (_a) {
...
-        });
-    }, [host, redirection]);
+        if (testMode === true) {
+            window.location.replace(redirection);
+        }
+        else {
+            BarongApiUtil.register(host, data)
+                .then(function (response) {
+                response.status === 201
+                    ? window.location.replace("" + redirection)
```
Comment (akhlopiachyi) [inline example/src/react-barong/index.js:100]: what is the purpose of `"" + redirection`
I guess using just `redirection` will be enough

File: example/src/react-barong/index.js [hunk 4]
```diff
@@ -114,4 +129,28 @@ var BarongRegisterForm = function (_a) {
...
+            BarongApiUtil.logout(host)
+                .then(function (response) {
+                if (response.status === 200) {
+                    window.location.replace(redirection);
+                }
+                else {
+                    window.console.error(response);
+                }
+            })
+                .catch(function (err) {
```
Comment (akhlopiachyi) [inline example/src/react-barong/index.js:148]: fix indentation

File: example/src/react-barong/index.js (hunk 1 above)
Comment (berezh) [reply example/src/react-barong/index.js:63]: this is the example project. The folder "react-barong" contains here the generated js code.

**Review by akhlopiachyi — APPROVED** [review]

**Pull request merged**